apps/studio/pages/projects.tsx

```diff
@@ -38,6 +38,7 @@ const ProjectsPage: NextPageWithLayout = () => {
 
   return (
     <>
+      <AlertError subject="Failed to retrieve organizations" />
```

Comment:
Forgotten test stuff for another feature? :)

Reply:
hahahaha god dang it 🤦